module-1/pandas-project/your-code/pandas-project.ipynb

```diff
@@ -16,23 +16,9 @@
   },
```

Comment:
1. Doing this is a really good practice. "Checking basic statistical info and info type", it clarifies a lot further steps of analysis. 
2. Thanks for using loc function!! 
3. In[33]: Do you mean Irak (the oldest country) population evolution, don't you? 
4. Remember to use head function always when dealing with dataframes. 
5. There are some values at the end of the analysis that are difficult to interpret and could sound "counter-intuitive". You say that you need to need review some values, maybe the global evaluation should be focused in a different way. I mean the code is perfect and it works but the output numbers could be difficult to explain in case they are not wrong. I would take a look on the bins throughout your notebook, It could be useful focus your reviews in the bining and dedicate a specific paragraph for analysis bins, both for local and global evaluation.

Reply:
Hello Belen, 
In the global evaluation I wanted to remark that this database has a lot of missing values and registers. This does not mean that isn't clean. No nan values, outliers or empty cells found.
Thank you for your feedback and time, it means a lot for my encoding improvement.